docs/docsite/rst/vmware/scenario_clone_template.rst

```diff
@@ -0,0 +1,124 @@
+.. _vmware_guest_from_template:
+
+****************************************
+Deploy a virtual machine from a template
+****************************************
+
+Introduction
+============
+
+This guide will show you how to utilize Ansible to clone a virtual machine from already existing VMware template.
+
+Scenario Requirements
+=====================
+
+* Software
+
+    * Ansible 2.5 or later must be installed
+
+    * The Python module *Pyvmomi* must be installed on the Ansible (or Target host if not executing against localhost)
+
+    * Installing the latest *Pyvmomi* via pip is recommended [as the OS packages are usually out of date and incompatible]
+
+* Hardware
+
+    * At least one standalone ESXi server or
+
+    * vCenter Server with at least one ESXi server
+
+* Access / Credentials
+
+    * Ansible (or the target server) must have network access to the either vCenter server or the ESXi server you will be deploying to
+
+    * Username and Password
+
+    * Administrator user with following privileges
+
+        - Virtual machine.Provisioning.Clone virtual machine on the virtual machine you are cloning
+        - Virtual machine.Inventory.Create from existing on the datacenter or virtual machine folder
+        - Virtual machine.Configuration.Add new disk on the datacenter or virtual machine folder.
+        - Resource.Assign virtual machine to resource pool on the destination host, cluster, or resource pool.
+        - Datastore.Allocate space on the destination datastore or datastore folder.
+        - Network.Assign network on the network to which the virtual machine will be assigned.
+        - Virtual machine.Provisioning.Customize on the virtual machine or virtual machine folder if you are customizing the guest operating system.
+        - Virtual machine.Provisioning.Read customization specifications on the root vCenter Server if you are customizing the guest operating system.
+
+Assumptions
+===========
+
+- All variable names and VMware object names are case sensitive
+- VMware allows creation of virtual machine and templates with same name across datacenters and within datacenters.
+- You need to use Python 2.7.9 version in order to use *validate_certs* option, as this version is capable of changing the SSL verification behaviours.
+
+Caveats
+=======
+
+- Hosts in the ESXI cluster must have access to the datastore that the template resides on.
```

Comment:
is it ESXI or ESXi? I see both in different places

Reply:
ESXi

Reply:
The first line under "Caveats" still says ESXI.